src/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "strict": false,
+    "strictNullChecks": false,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "noImplicitReturns": false,
+    "noImplicitAny": false,
+    "noImplicitThis": false,
+    "noFallthroughCasesInSwitch": false,
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "incremental": true,
+    "sourceMap": true,
+    "inlineSourceMap": false,
+    "inlineSources": true,
+    "preserveSymlinks": true,
+    "removeComments": true,
+    "jsx": "react",
+    "lib": ["ES2018", "dom"],
+    "target": "ES2018",
+    "allowJs": true,
```

Comment:
You've converted the one and only Javascript source file in this repo; why is this true?

Reply:
Etch is written in JavaScript

Reply:
Oof I mixed this up with checkJs. Sounds good 👍